src/main/java/uk/gov/digital/ho/hocs/casework/contributions/ContributionReceivedStages.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.digital.ho.hocs.casework.contributions;
+
+public enum ContributionReceivedStages {
+    MPAM_TRIAGE,
+    MPAM_TRIAGE_ESCALATE,
+    MPAM_DRAFT,
+    MPAM_DRAFT_ESCALATE;
+
+    public static boolean contains(String stage) {
```

Comment:
Could probably pull this out into a seperate helper class as repeated, only minor though so not blocking.

Reply:
It didn't really need two enums, so merged them together.